.github/workflows/deploy-to-control-plane.yml

```diff
@@ -5,9 +5,8 @@ name: Deploy-To-Control-Plane
 # Controls when the workflow will run
 on:
   # Uncomment the lines you want actions that will cause the workflow to Triggers the workflow on push or pull request events but only for the main branch
-
-  # push:
-  #   branches: [main]
+  push:
+    branches: [main]
```

Comment:
Did it work?

We don't `main` branch. We have `master`.

Reply:
Okay. It got fixed in https://github.com/shakacode/react-webpack-rails-tutorial/commit/4890a544982c1999f88b642d70c687ae95ba0931